source/codegen/metadata/nidaqmx/enums_addon.py

```diff
@@ -5,6 +5,15 @@
 }
 
 enums_validation_suppressions = {
```

Comment:
Consider: Is there a helpful comment that could be added here? I know, in general, the DAQ metadata is used for more than just grpc so contains fields/enums/etc that grpc does not use. Is there anything special about these new fields that helps understand why they are in the metadata but not used by functions or attributes?

Reply:
Yes. Here are some suggestions for line comments.

CouplingTypes: Overlaps with Coupling1 and Coupling2

DAQmxErrors and DAQmxWarnings: Used by nidaqmx-python

FillMode: Same as GroupBy

Impedance1: Should be a mapped enum, but grpc-device did not support those when this was added and adding it now would break compatibility

Save: Save as SaveOptions

SignalModifiers: Same as InvertPolarity

TaskMode: Same as TaskControlAction

TermCfg: Overlaps with InputTermCfg, InputTermCfg2, etc.

TriggerUsageTypes: Overlaps with TriggerType4-10

WDTTaskAction: Same as WatchdogControlAction